apps/api/test/seeders/validator.seeder.ts

```
@@ -16,7 +16,13 @@ export class ValidatorSeeder {
       maxRate: input.maxRate || faker.number.int({ min: 0, max: 10000000 }),
       maxChangeRate: input.maxChangeRate || faker.number.int({ min: 0, max: 10000000 }),
       minSelfDelegation: input.minSelfDelegation || faker.number.int({ min: 0, max: 10000000 }),
+      keybaseUsername: input.keybaseUsername || faker.internet.userName(),
       keybaseAvatarUrl: input.keybaseAvatarUrl || faker.image.avatar()
     };
   }
+
+  static async createInDatabase(overrides: Partial<CreationAttributes<Validator>> = {}): Promise<Validator> {
+    const seed = this.create(overrides);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)